src/wincameranode.cpp

```diff
@@ -37,8 +42,17 @@ int main(int argc, char** argv)
         }
         else
         {
+            int i = 0;
+#ifdef INTERACTIVE
+            for (auto sym : ros_win_camera::WindowsMFCapture::EnumerateCameraLinks(false))
+            {
+                std::wcout << i++ << " - " << sym.c_str();

```

Comment:
Recommend using the ROS_INFO macro.

Reply:
I was only worried about the case where logger levels were below "Info" and INTERACTIVE preprocessor was defined. I have now added appropriate conditions.